.github/workflows/release.yml

```diff
@@ -81,22 +88,17 @@ jobs:
         id: extract-branch
         run: echo "::set-output name=branch::$(echo ${GITHUB_REF#refs/heads/})"
 
-      - name: Get Next Bump
-        id: next-bump
-        run: |
-          echo "::set-output name=bump::$(node node_modules/conventional-recommended-bump/cli.js -p angular)"
-          echo ${{steps.next-bump.outputs.bump}}
-
-      ## On the master branch, we'll do a patch bump on every commit. This will create a commit with
-      ## of a push action to allow a PAT to bypass branch protections (pushing directly on the
-      ## branch).
+      ## On the master branch, we'll do a patch bump on every commit by default.
+      ## However, a `bump-level` can be specified with 'minor' or 'major' if needed.
+      ## This will create a commit with of a push action to allow a PAT to bypass
+      ## branch protections (pushing directly on the branch).
       - name: Lerna Bump
-        if: "!inputs.version"
-        run: yarn bump --yes ${{steps.next-bump.outputs.bump}}
+        if: '!inputs.version'
```

Comment:
hmm. This is a weird linting update. I wonder why my `yml` settings are different.